ava-labs/core-extension
# fix: treat BTC as known network

## Description
* [CP-9154](https://ava-labs.atlassian.net/browse/CP-9154)

## Changes
* When network is already supported by the extension, do not use the request's payload, but rather resolve the network object via `NetworkService` and use that instead.

## Testing
1. Open extension, switch to C-Chain, go to Send
2. Connect to Core Web
3. Switch to Bitcoin _**via Core Web**_
4. Open extension again, you should not see `Unknown error`

## Screenshots:
### Before

https://github.com/user-attachments/assets/e8f54b13-4318-47ec-abf0-f49eaaf2e601

### After

https://github.com/user-attachments/assets/685504c9-72c0-4fe7-b124-b0e0f24ba267




## Checklist for the author
